base/complex.jl

```diff
@@ -541,6 +541,38 @@ function cis(z::Complex)
     Complex(v * c, v * s)
 end
 
+function cispi(sign::Bool)
+    sign ? -1 : 1
```

Comment:
These seem like they could all be one-liners.

Reply:
Done.